### PR TITLE
Multi-mission radioisotope thermoelectric generator knowledge

### PR DESCRIPTION
**Pull request by jmtejal**
#839

### CODE REVIEW

File: knowledge/science/astronomy_tech/radioisotope_termo_generator/attribution.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+created_by:
```
Comment (bjhargrave) [inline knowledge/science/astronomy_tech/radioisotope_termo_generator/attribution.txt:1]: Please use a valid attribution file. This is just a copy of the qna.yaml.

File: knowledge/science/astronomy_tech/radioisotope_termo_generator/qna.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  question:  When did the Department of Energy (DOE) awarded the MMRTG contract to a team led by Aerojet Rocketdyne
+- answer: | MMRTG
+  question: What is powered by eight Pu-238 dioxide general-purpose heat source (GPHS) modules, provided by the US Department of Energy (DOE). Initially, these eight GPHS modules generate about 2 kW thermal power.
+- answer: | Curiosity
+  question: What is the MSL rover that was successfully landed in Gale Crater on August 6, 2012
+- answer: |  US$109,000,000
+  question: How many million dollars did it cost to produce and deploy MMRTG
+- answer: | Nasa Dragonfly
+  question: Which mission to Saturn's moon Titan will use one of the two MMRTGs for which the Aerojet Rocketdyne/Teledyne Energy Systems team has recently received a contract
+task_description: ' '
```
Comment (bjhargrave) [inline knowledge/science/astronomy_tech/radioisotope_termo_generator/qna.yaml:14]: Please provide a valid task description.

File: knowledge/science/astronomy_tech/radioisotope_termo_generator/qna.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+created_by:
```
Comment (bjhargrave) [inline knowledge/science/astronomy_tech/radioisotope_termo_generator/qna.yaml:1]: This value must be your github id.

**Review by bjhargrave — CHANGES_REQUESTED** [review]

**Comment by bjhargrave** [comment]
Also, the folder name has a misspelling that should be corrected.

`knowledge/science/astronomy_tech/radioisotope_thermo_generator`

**Comment by jjasghar** [comment]
Closing due to combined PR here:  #1142

**Pull request closed without merging**